npm/webpack-dev-server/README.md

```diff
@@ -68,6 +68,7 @@ We then merge the sourced config with the user's webpack config, and layer on ou
 | --------------------------- | ------- |
 | <= v1                       | <= v9   |
 | >= v2                       | >= v10  |
+| >= v4                       | >= v13  |
```

Comment:
@fochlac hey! We are looking to merge this soon. Should this be v3?

Also can you rebase again? Thank you!

If you can't, let me know and I can try to pick it up.

Reply:
Hi, only saw your comment just now. I think the new version should be v4 as we are currently on v3, and for full compatibility we need dev-server >=v4 and cypress >=v13.
We could consider adapting this like this as well for more clarity:
| --------------------------- | ------- |
| <= v1                       | <= v9   |
| v2 - v3                      | v10 - v12  |
| >= v4                       | >= v13  |